.github/workflows/test-linux.yml

```diff
@@ -65,6 +65,7 @@ jobs:
             tibble
             haven
             R6
+            survival
```

Comment:
I added survival here, despite it being (because it is?) such a problem child so we can write tests for this especially weird/bad case.